src/EnumServiceProvider.php

```diff
@@ -24,4 +32,31 @@ public function register()
             'command.make:enum',
         ]);
     }
+
+    public function bootValidationRules(): void
+    {
+        Validator::extend('enum', function (string $attribute, $value, array $parameters, ValidatorContract $validator): bool {
+            $enum = $parameters[0] ?? null;
+
+            return (new Enum($enum))->passes($attribute, $value);
```

Comment:
For clarity, I'd name these classes `EnumRule`, `EnumIndexRule` etc.

Reply:
https://github.com/spatie/laravel-enum/pull/13/commits/d28b7f670dbdef217afb9c933f7b99c785063653